### PR TITLE
Pods 9780

### PR DESCRIPTION
**Pull request by nonebula**
Phase 1 warn log clean-up.

### CODE REVIEW

**Review by jamdade-sharad — APPROVED** [review]
lgtm

File: app/audit/AuditService.scala [hunk 1]
```diff
@@ -71,7 +71,7 @@ class AuditServiceImpl @Inject()(
         logger.debug(s"[AuditService][sendEvent] successfully sent ${event.auditType}")
 
       case Failure(e) =>
-        logger.error(s"[AuditService][sendEvent] failed to send event ${event.auditType}", e)
+        logger.warn(s"[AuditService][sendEvent] failed to send event ${event.auditType}", e)
```
Comment (PavelVjalicin) [inline app/audit/AuditService.scala:74]: Audit failures should be errors. They are critical to get right, they store historic data to be retrieved if something goes wrong.

File: app/models/TolerantAddress.scala [hunk 2]
```diff
@@ -120,7 +120,7 @@ object TolerantAddress {
     val addressLines: (Option[String], Option[String], Option[String], Option[String]) = {
       lines.size match {
         case 0 =>
-          logger.warn(s"[NoAddressLinesFoundException]-$postCode,$countryCode")
+          logger.info(s"[NoAddressLinesFoundException]-$postCode,$countryCode")
```
Comment (PavelVjalicin) [inline app/models/TolerantAddress.scala:123]: This seems important. Warn was fine.

File: app/utils/RetryHelper.scala [hunk 3]
```diff
@@ -49,7 +49,7 @@ trait RetryHelper  {
           val call: Callable[Future[Int]] = new Callable[Future[Int]](){
             def call(): Future[Int] = Future.successful(1)
           }
-          logger.warn(s"Failure, retrying after $wait ms, attempt $currentAttempt")
+          logger.info(s"Failure, retrying after $wait ms, attempt $currentAttempt")
```
Comment (PavelVjalicin) [inline app/utils/RetryHelper.scala:52]: This might be useful, so people know that initial requests failed. Should be warn.

**Pull request merged**